### DISCUSSION TITLE
How to create cover or light entities which would then be generated in HA?

### DISCUSSION THREAD

**Discussion by nabbl** [zachowj/node-red-contrib-home-assistant-websocket#601] [Q&A]
So currently in the entity node you can define switches, binary sensors or sensors.
What I want to do is add all my enocean lights and covers to HA by using the home assistant entity nodes.

How would I do this correctly? I cant create an entity which would be like "cover.living_room_cover".

I am probably doing something wrong here.

**Comment by zachowj** [comment]
Currently, the light domain is not supported as an entity from NR.
